docs/sidebars.js

```diff
@@ -1,15 +1,14 @@
 module.exports = {
   docs: [
     'index',
-    'user-guide',
     {
       API: ['api/http', 'api/websocket', 'api/webhook'],
     },
     {
       Sources: [
-        'sources/chat-plugin',
         'sources/facebook',
         'sources/google',
+        'sources/chat-plugin',
         'sources/sms-twilio',
         'sources/whatsapp-twilio',
       ],
```

Comment:
This must have been a faulty rebase, can you please move the chat plugin back at the top of the list (as there's no real relevant to the order, defaulting to alphabetical is a sensible choice)